Project2015To2017/ProjectConverter.cs

```diff
@@ -125,7 +133,7 @@ IProgress<string> progress
 			progress.Report("Solution parsing started.");
 			var solution = SolutionReader.Instance.Read(target, progress);
 
-			if (solution?.ProjectPaths == null)
+			if (solution.ProjectPaths == null)
```

Comment:
It really can't be null (instance always created).

Reply:
Yes, if only we had nullable reference types...